packages/core/admin/admin/src/StrapiApp.tsx

```diff
@@ -629,6 +633,15 @@ class StrapiApp {
       [
         {
           path: '/*',
+          errorElement: (
+            <Provider store={store}>
```

Comment:
Don't we want to embed this a bit more in the Admin panel UI instead ? => for later

Reply:
Do you mean to remove the need for adding the extra providers? I think we could look at moving it down the router a bit, it'd be nice to actually clean that part of the app up to be honest as well.

Reply:
Yeah mostly want it to appear but want to see the rest of the UI below for ex. + makes the providers stack used once only :)